Add 'aggregates' and 'countries_only' options

bapfeld opened the pull request:
The wbstats package is awesome!  There are times when I want all countries but no aggregates and I figured others might be interested in that feature as well.

jpiburn commented:
Brendan,
thanks for the pull request. I added in your enhancements and updated wb_cachelist. I am going to resubmit to CRAN soon. I appreicate you taking the time to add those features. I modified your original code slightly to avoid some corner case warning messages that would probably never happen but worth avoiding none the less.

Thanks again

bapfeld commented:
Great, glad to contribute!

> On Sep 20, 2016, at 3:27 PM, Jesse Piburn <notifications@github.com> wrote:
> 
> Brendan,
> thanks for the pull request. I added in your enhancements and updated wb_cachelist. I am going to resubmit to CRAN soon. I appreicate you taking the time to add those features. I modified your original code slightly to avoid some corner case warning messages that would probably never happen but worth avoiding none the less.
> 
> Thanks again
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/GIST-ORNL/wbstats/pull/1#issuecomment-248423782>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AQiYA7h03Ys-dy36aNNuae18kEwHKt3kks5qsEG4gaJpZM4KBGVN>.
>
